curvefs/src/metaserver/trash.cpp

```diff
@@ -129,7 +140,7 @@ bool TrashImpl::NeedDelete(const TrashItem &item) {
     // if fs recycleTimeHour is not 0, return true
     uint64_t recycleTimeHour = GetFsRecycleTimeHour(item.fsId);
     if (recycleTimeHour == 0) {
-        return ((now - item.dtime) >= options_.expiredAfterSec);
+        return ((now - item.dtime) >= FLAGS_trash_expiredAfterSec);
```

Comment:
if so, should `options_.expiredAfterSec` be removed?

Reply:
https://github.com/opencurve/curve/blob/feat/curvefs/change_conf_on_fly/curvefs/src/metaserver/trash.cpp#L44-L49

Reply:
> if so, should `options_.expiredAfterSec` be removed?

The reason for retaining this is to consider that the configuration-related content will be rewritten later. If you keep it, you can know that there is such a configuration item.